src/utils.js

```diff
@@ -32,4 +32,63 @@ H5P.Components.utils.createElement = (tag, options, style = {}) => {
   Object.assign(element.style, style);
 
   return element;
-};
\ No newline at end of file
+};
+
+/** @constant {number} DEFAULT_LINE_HEIGHT Default line height when it is "normal" */ 
+const DEFAULT_LINE_HEIGHT = 1.2;
```

Comment:
It is good practice to put instance variables/constants at the beginning of a class.